Lib/test/test_keyword.py

```diff
@@ -21,7 +22,10 @@ def test_changing_the_kwlist_does_not_affect_iskeyword(self):
         self.assertFalse(keyword.iskeyword('eggs'))
 
     def test_all_keywords_fail_to_be_used_as_names(self):
-        for key in keyword.kwlist:
+        all_keywords = set(keyword.kwlist)
+        if use_old_parser():
+            all_keywords -= {'__new_parser__'}
```

Comment:
I would use discard(), but your code is correct ;-)

```suggestion
            all_keywords.discard('__new_parser__')
```

Reply:
> I would use discard(), but your code is correct ;-)

I like `discard()` :)